format/spec.md

```diff
@@ -693,6 +686,64 @@ A snapshot's `first-row-id` is assigned to the table's current `next-row-id` on
 The snapshot's `first-row-id` is the starting `first_row_id` assigned to manifests in the snapshot's manifest list.
 
 
+#### Snapshot Summary
+
+##### Required Field `operation`
+The snapshot summary's `operation` field is used by some operations, like snapshot expiration, to skip processing certain snapshots. Possible `operation` values are:
+
+*   `append` -- Only data files were added and no files were removed.
+*   `replace` -- Data and delete files were added and removed without changing table data; i.e., compaction, changing the data file format, or relocating data files.
+*   `overwrite` -- Data and delete files were added and removed in a logical overwrite operation.
+*   `delete` -- Data files were removed and their contents logically deleted and/or delete files were added to delete rows.
+
+##### Optional Metrics
+All metrics fields should have numeric string values (e.g., `"120"`).
```

Comment:
I think we should note that these values may be used by engines for optimizations so they must be correct but they can be skipped if an engine doesn't want to write them. Maybe that's clear enough already though with "optional"